srl/src/main/java/edu/illinois/cs/cogcomp/srl/utilities/WeightVectorUtils.java

```diff
@@ -0,0 +1,134 @@
+/**
```

Comment:
I think my point is different: if you look at the file, it contains functions from both core-utils and SL. Moving this file to one of those projects require one of them to be dependency of the other.